fiaas_deploy_daemon/deployer/deploy.py

```diff
@@ -74,14 +74,14 @@ def _update(self, app_spec):
                 self._bookkeeper.success(app_spec)
             LOG.info("Completed deployment of %r", app_spec)
         except Exception:
+            LOG.exception("Error while deploying %s: ", app_spec.name)
```

Comment:
is the message of the exception logged? might be useful to have as well?

Reply:
`logging.exception` automatically includes the stack-trace when it's inside an `except`

Reply:
ah I see, then it's ok. Just a comment: not sure if it's doable, but do you think it makes sense to use "error" then instead with the actual exception message rather than the full stack trace so the user is presented with something not scary? Then we can leave the log.exception after the bookeeping for internal/operator debugging.

Reply:
I think changing what is logged to account for that would be a little odd, and the intention could easily be missed. So I've changed the formatter that's used for the captured logs, so it skips the trace and includes just the last line. It leaves the logs to stdout untouched.